notification/src/main/java/com/kyriba/conference/notification/domain/EmailNotification.java

```diff
@@ -0,0 +1,58 @@
+package com.kyriba.conference.notification.domain;
+
+import com.kyriba.conference.notification.domain.dto.EmailMessageOptions;
+import com.kyriba.conference.notification.domain.dto.EmailNotificationMessage;
+import com.kyriba.conference.notification.domain.dto.MessageType;
+import lombok.*;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.Set;
+
+@Entity
+@Table(name = "email_notification",
```

Comment:
Useless information if only you are not going to generate schema from JPA mapping.